examples/kubernetes/slos/apiserver-request-errors.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: pyrra.dev/v1alpha1
+kind: ServiceLevelObjective
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: apiserver-read-response-errors
+  namespace: monitoring
+spec:
+  description: ""
+  indicator:
+    ratio:
+      errors:
+        metric: apiserver_request_total{job="apiserver",verb=~"LIST|GET",code=~"5.."}
+      total:
+        metric: apiserver_request_total{job="apiserver",verb=~"LIST|GET"}
+  target: "99.99"
+  window: 2w
+---
+apiVersion: pyrra.dev/v1alpha1
+kind: ServiceLevelObjective
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: apiserver-write-response-errors
+  namespace: monitoring
+spec:
+  description: ""
+  indicator:
+    ratio:
+      errors:
+        metric: apiserver_request_total{job="apiserver",verb=~"POST|PUT|PATCH|DELETE",code=~"5.."}
+      total:
+        metric: apiserver_request_total{job="apiserver",verb=~"POST|PUT|PATCH|DELETE"}
```

Comment:
You might want to add `,scope=~"resource|"` here to, so that `healthz` requests would not be included